src/components/ModalPayment/ModalAmount/ModalAmount.tsx

```diff
@@ -75,18 +100,18 @@ export const Modal = (props: Props) => {
         <SelectAmtContainer>
           {buttonAmounts.map((amount) => (
             <button
-              key={amount.text}
+              key={`$${amount}`}
               type="button"
               className={
-                selected === amount.text
+                selectedAmount === `$${amount}`
```

Comment:
Are `amount` and `selectedAmount` are both numbers (but string types)? If so i think `selectedAmount === amount` should be fine otherwise I think the buttons won't change color when clicked on. Can double check though

Reply:
@billy-yuan yeah good callout; I changed it to compare the amount value now instead of the 'pretty string'. (I mean they're still strings though, but yeah.)